packages/panzer/mini-em/example/BlockPrec/main.cpp

```diff
@@ -642,12 +642,12 @@ void createExodusFile(const std::vector<Teuchos::RCP<panzer::PhysicsBlock> >& ph
     mesh->getElementBlockNames(block_names);
 
     Teuchos::ParameterList output_pl("Output");
-    Teuchos::ParameterList& cell_avg_q = output_pl.sublist("Cell Average Quantities");
+    // Teuchos::ParameterList& cell_avg_q = output_pl.sublist("Cell Average Quantities");
     Teuchos::ParameterList& cell_avg_v = output_pl.sublist("Cell Average Vectors");
     cell_avg_v.set(block_names[0],"CURRENT");
-    Teuchos::ParameterList& cell_q = output_pl.sublist("Cell Quantities");
-    Teuchos::ParameterList& nodal_q = output_pl.sublist("Nodal Quantities");
-    Teuchos::ParameterList& a_nodal_q = output_pl.sublist("Allocate Nodal Quantities");
+    // Teuchos::ParameterList& cell_q = output_pl.sublist("Cell Quantities");
+    // Teuchos::ParameterList& nodal_q = output_pl.sublist("Nodal Quantities");
+    // Teuchos::ParameterList& a_nodal_q = output_pl.sublist("Allocate Nodal Quantities");
```

Comment:
Do you want these lines commented out or just removed?  We can always get them back via git if we need to.